### ISSUE TITLE
This program cannot be used in production at all.

### ISSUE THREAD

**Issue by Pengree** [chidiwilliams/buzz#414]
If you select the "transcribe" mode, "tiny" model, and English language, the real-time transcription speed is slightly slow but still acceptable, and the recognition rate is also acceptable. 

However, if you choose other models (base, small, medium), the real-time transcription speed is too slow to be achieved in real-time, and if you select other languages,  for example, Chinese, the recognition rate is also very low. 

In short, this software cannot be used in production at all. Charging $9.9 on App Store for this software is very unreasonable and misleading to users.

**Comment by chidiwilliams** [comment]
Hi, I'm sorry to hear about the issues with the real-time transcription. I tried to make it clear [on the docs](https://github.com/chidiwilliams/buzz#readme) that the transcription is resource-intensive and may not keep up with real-time. I'll make it clearer on the website and App Store listing that the app is best used for file imports or real-time with the tiny model.

Unfortunately, the App Store does not let me offer you a refund directly, but you can request one [here](https://reportaproblem.apple.com/). My apologies for your experience with the app once again.

**Comment by ttimasdf** [comment]
At least it works for you, for me the transcription won't start for no reason. By the way you could try Windows [live caption](https://support.microsoft.com/en-us/windows/use-live-captions-to-better-understand-audio-b52da59c-14b8-4031-aeeb-f6a47e6055df). It takes only 10% CPU consumption, no GPU usage, fully offline, English only, super powerful (and works for me 😂).

**Comment by chidiwilliams** [comment]
@ttimasdf Are you able to transcribe files?

**Comment by Pengree** [comment]
> Hi, I'm sorry to hear about the issues with the real-time transcription. I tried to make it clear [on the docs](https://github.com/chidiwilliams/buzz#readme) that the transcription is resource-intensive and may not keep up with real-time. I'll make it clearer on the website and App Store listing that the app is best used for file imports or real-time with the tiny model.
> 
> Unfortunately, the App Store does not let me offer you a refund directly, but you can request one [here](https://reportaproblem.apple.com/). My apologies for your experience with the app once again.

@chidiwilliams I wanted to take a moment to thank you for your prompt response to my previous problem. Your clarification on the real-time transcription issue was helpful, and I appreciate your effort to make it clearer on the website and App Store listing. I understand that the transcription is resource-intensive and may not keep up with real-time when using certain models and languages.

Furthermore, I would like to apologize for my previous  tone, which may have been too harsh. After further testing and consideration, I realized that your software's functionality is impressive, particularly the transcription and translation features when importing files.

I am willing to continue using your software as a loyal customer, and I don't require a refund. I would appreciate it if you could keep me updated on any updates or improvements to the software in the future.

Thank you again for your response, and I wish your software continued success.

**Comment by ttimasdf** [comment]
> @ttimasdf Are you able to transcribe files?

Yes, file transcription is good. When using live transcription, the visualizer indicates my mic is working, but CPU and GPU is low which leads me to believe that the model is not running. Weird it is

**Comment by chidiwilliams** [comment]
@Pengree, no problem at all. I'll update this repository as new updates come out. Thanks!
